config/routes.rb

```diff
@@ -1,5 +1,4 @@
 Rails.application.routes.draw do
   mount DDS::Base, at: '/'
   get "apiexplorer" => "swaggerui#index"
-  root to: redirect("/portal", status: 302)
```

Comment:
I created a Bug issue in Jira. We can leave this PR open until the fix is applied to develop.